test/integration/authrep/list_app_keys_test.rb

```diff
@@ -0,0 +1,131 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../test_helper')
```

Comment:
This is basically the same as the other file, check if my comments apply to this one too.

Reply:
I think they do.